allow customize buck2 and python path

cccclai opened the pull request:
Differential Revision: D54078165

For my development flow, I'd need to have buck2 and python3 in a different path. Modify the script to allow us to customize the path

Test with 
```
sh backends/qualcomm/scripts/build.sh
```

chunit-quic commented on backends/qualcomm/scripts/build.sh:
```diff
@@ -30,6 +30,14 @@ BUILD_AARCH64="true"
 CMAKE_AARCH64="build_android"
 CLEAN="true"
 
+if(NOT PYTHON_EXECUTABLE)
+  set(PYTHON_EXECUTABLE python3)
+endif()
+
+if(NOT BUCK2)
+  set(BUCK2 buck2)
+endif()
```
Bash sematic should be this
```bash
if [ -z PYTHON_EXECUTABLE ]; then
  PYTHON_EXECUTABLE="python3"
fi
 
if [ -z BUCK2 ]; then
  BUCK2="buck2"
fi
```

cccclai replied:
Oh good catch. Update with the comments. What do you think?

chunit-quic replied:
LTGM. Thanks a lot for helping!

cccclai committed:
allow customize buck2 and python path (#2044)

Summary:

For my development flow, I'd need to have buck2 and python3 in a different path. Modify the script to allow us to customize the path

Test with 
```
sh backends/qualcomm/scripts/build.sh
```

Reviewed By: kirklandsign

Differential Revision: D54078165
